Assets/Scripts/GameAudio/AmbientAudioController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using GameTime;
+using Managers;
+using UnityEngine;
+
+namespace GameAudio
+{
+    public class AmbientAudioController : MonoBehaviour
+    {
+        [SerializeField][Tooltip("Clock which is managing the game time ")]
+        private Clock clock;
+
+        private void OnEnable()
+        {
+            Clock.SunRise += DayAmbience;
+            Clock.SunSet += NightAmbience;
+            Clock.MoonRise += MoonRiseAmbience;
+            Clock.MoonSet += MoonShineAmbienceStop;
+        }
+
+        private void OnDisable()
+        {
+            Clock.SunRise -= DayAmbience;
+            Clock.SunSet -= NightAmbience;
+            Clock.MoonRise -= MoonRiseAmbience;
+            Clock.MoonSet -= MoonShineAmbienceStop;
+        }
+
+        private void Start()
+        {
+            AudioManager.Instance.FadeIn(clock.IsNight ? "AmbCrickets" : "AmbBirds", 5);
+        }
+
+        private void DayAmbience()
+        {
+            AudioManager.Instance.FadeIn("AmbBirds",120);
+        }
+
+        private void NightAmbience()
+        {
+            AudioManager.Instance.FadeOut("AmbBirds",10);
```

Comment:
Maybe `FadeOutCurrent `and/or `FadeOutAll`?

Reply:
Because of the sleep event? I'll implement it, but it will stop every sound managed by audiomanager. So if there's music playing in the background it'll stop that aswell.

Reply:
fadeout everything with AudioManager.Instance.FadeOutPlaying(float duration)